lib/WebpackConfig.js

```diff
@@ -157,6 +158,10 @@ class WebpackConfig {
         this.styleEntries.set(name, src);
     }
 
+    addLoader(loader) {
+        this.loaders.push(loader);
+    }
+
```

Comment:
We need to expose this in `index.js` as well (that file/module holds the true public API). And I think we should (in that file) create `addLoader()` and also `addRule()` (which would just call `addLoader()`)